angular.json

```diff
@@ -24,13 +24,13 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.css",
               "./node_modules/quill/dist/quill.core.css",
               "./node_modules/quill/dist/quill.bubble.css",
               "./node_modules/quill/dist/quill.snow.css",
               "./node_modules/quill-emoji/dist/quill-emoji.css",
               "./node_modules/quill-mention/dist/quill.mention.min.css",
-              "./node_modules/katex/dist/katex.min.css"
+              "./node_modules/katex/dist/katex.min.css",
+              "src/styles.scss"
```

Comment:
Moving `styles.scss` to the bottom to allow for cleaner override of any library styles